tests/test_htmIndex.py

```diff
@@ -48,27 +47,12 @@
 
 REGENERATE_COMPARISON = False  # Regenerate comparison data?
 
+
 def make_coord(ra, dec):
     """Make an ICRS coord given its RA, Dec in degrees."""
     return afwCoord.IcrsCoord(afwGeom.Angle(ra, afwGeom.degrees), afwGeom.Angle(dec, afwGeom.degrees))
 
 
-def makeWcs(ctr_coord, ctr_pix=afwGeom.Point2D(2036., 2000.),
-            pixel_scale=2*afwGeom.arcseconds, pos_angle=afwGeom.Angle(0.0)):
-    """Make a simple TAN WCS
-
-    @param[in] ctr_coord  sky coordinate at ctr_pix
-    @param[in] ctr_pix  center pixel; an lsst.afw.geom.Point2D; default matches LSST
-    @param[in] pixel_scale  desired scale, as sky/pixel; an lsst.afw.geom.Angle; default matches LSST
-    @param[in] pos_angle  orientation of CCD w.r.t. ctr_coord, an lsst.afw.geom.Angle
-    """
-    pos_angleRad = pos_angle.asRadians()
-    pixel_scaleDeg = pixel_scale.asDegrees()
-    cdMat = np.array([[math.cos(pos_angleRad), math.sin(pos_angleRad)],
-                      [-math.sin(pos_angleRad), math.cos(pos_angleRad)]], dtype=float) * pixel_scaleDeg
-    return lsst.afw.image.makeWcs(ctr_coord, ctr_pix, cdMat[0, 0], cdMat[0, 1], cdMat[1, 0], cdMat[1, 1])
-
-
```

Comment:
Because the local function `makeWcs` was completely redundant with `afw.geom.makeSkyWcs`. I've tried to weed that sort of thing out as part of this ticket. I found a surprising number of variants on that private function scattered throughout our code.